src/core_modules/capture-core/components/Pages/common/EnrollmentOverviewDomain/EnrollmentPageLayout/EnrollmentPageLayout.js

```diff
@@ -10,6 +10,7 @@ import { DefaultPageTitle, EnrollmentPageKeys } from './DefaultEnrollmentLayout.
 
 const getEnrollmentPageStyles = () => ({
     container: {
+        minHeight: '90vh',
```

Comment:
Open for ideas on this. The page only took up the size of the widgets, but that might not have been the full size. Also, App-platform adds a small padding at the bottom, so setting it to 90 instead of 100 means it will work as corrected.

Reply:
Seems to work and I don't have a better suggestion.